proto/serverless/proto/instrumentation/tagset/v1/tagset.proto

```diff
@@ -0,0 +1,272 @@
+syntax = "proto3";
+
+package serverless.proto.instrumentation.tagset.v1;
+
+message Tags {
+  // The required set of tags for The Serverless Ingest Platform. Payloads without all these tags will be rejected.
+  SlsSdkTags sls = 1;
+
+  // ============================================ //
+  // Defined TagSets start at field number 4000  //
+  // ========================================== //
+
+  // The root AWS Lambda Span tags.
+  optional AwsLambdaTags aws_lambda_tags = 4000;
+
+  // The AWS Lambda Initialization tags.
+  optional AwsLambdaInitializationTags aws_lambda_initialization_tags = 4001;
+
+  // The AWS Lambda Invocation tags.
+  optional AwsLambdaInvocationTags aws_lambda_invocation_tags = 4002;
+
+  // The AWS SDK Tags.
+  optional AwsSdkDynamodbTags aws_sdk_dynamodb_tags = 4003;
+  optional AwsSdkSqsTags aws_sdk_sqs_tags = 4004;
+  optional AwsSdkSnsTags aws_sdk_sns_tags = 4005;
+
+
+  // The HTTP Tags.
+  optional HttpTags http_tags = 4006;
+
+  // The HTTPS Tags.
+  optional HttpTags https_tags = 4007;
+}
+
+message SlsSdkTags {
+  // A Serverless Platform OrgId.
+  string org_id = 1;
+  // The Name of the Serverless SDK used to instrument.
+  string sdk_name = 2;
+  // The version of the Serverless SDK used to instrument.
+  string sdk_version = 3;
+  // The platform that was instrumented. Currently Lambda is the only supported platform.
+  string platform = 4;
+  // The service that was instrumented. For Lambda this will be the function name by default.
+  string service = 5;
+  // The region that instrumentation was performed in. This is used to determine which Serverless Ingest API to use.
+  string region = 6;
+
+  // An optional environment that can be attached.
+  optional string environment = 16;
+  // An optional namespace that can be attached.
+  optional string namespace = 17;
+}
+
+message AwsTags {
```

Comment:
Was it intentional that this set of tags was not added to the main `Tags` message? 🤔

Reply:
Nope, just forgot them, I'll add them.